tests/unit/dataactvalidator/test_a22_appropriations.py

```diff
@@ -1,12 +1,18 @@
 from dataactcore.models.stagingModels import Appropriation
 from dataactcore.models.domainModels import SF133
-from tests.unit.dataactvalidator.utils import number_of_errors
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
 
 
 _FILE = 'a22_appropriations'
 _TAS = 'a22_appropriations_tas'
 
 
+def test_column_headers(database):
+    expected_subset = {'row_number', 'obligations_incurred_total_cpe', 'expected_value_GTAS SF133 Line 2190'}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
```

Comment:
I see the multi-line description comments for `test_success` and `test_failure` were updated for most of these files but not this one.